bugbear.py

```diff
@@ -641,6 +643,19 @@ def is_abstract_decorator(expr):
 
         self.errors.append(B024(node.lineno, node.col_offset, vars=(node.name,)))
 
+    def check_for_b026(self, call: ast.Call):
+        try:
+            starred = next(arg for arg in call.args if isinstance(arg, ast.Starred))
+        except StopIteration:
+            return
+
+        if any(
+            (keyword.value.lineno, keyword.value.col_offset)
+            < (starred.lineno, starred.col_offset)
+            for keyword in call.keywords
+        ):
```

Comment:
I don't know if `call.keywords` is ordered by occurrence in the definition. If yes, we can also get away with only checking the first element

```suggestion
        if (call.keywords[0].value.lineno, call.keywords[0].value.col_offset) < (starred.lineno, starred.col_offset):
```

Reply:
I've checked against all tests in this repo and the keywords are indeed ordered by occurrence.